### ISSUE TITLE
Re-applying after closing and opening

### ISSUE THREAD

**Issue by Tovitas** [spicetify/spicetify-cli#2165]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version:  1.2.4
- Spicetify version:2.16.1
-CPU: AMD Ryzen 7 Pro 1700X Eight-Core Processor
-GPU: NVIDIA GeForce RTX 2060
-RAM: 32 GB
- OS: Windows 10


### 📝 Description

In short: Everytime I close Spotify and reopen it, I have to re-apply spicefy with powershell.. Is this common?

### 📸 Screenshots

_No response_

**Comment by kyrie25** [comment]
Duplicate of #1557

**Comment by kyrie25** [comment]
This happens when you use the Microsoft Store version of Spotify.
Uninstall then download Spotify from their website.
